config/database.php

```diff
@@ -41,11 +41,11 @@
 
         'mysql' => [
             'driver' => 'mysql',
-            'host' => env('DB_HOST', '127.0.0.1'),
+            'host' => env('localhost', 'localhost'),
             'port' => env('DB_PORT', '3306'),
-            'database' => env('DB_DATABASE', 'forge'),
-            'username' => env('DB_USERNAME', 'forge'),
-            'password' => env('DB_PASSWORD', ''),
+            'database' => env('prasmi', 'prasmi'),
+            'username' => env('root', 'root'),
+            'password' => env('Azerty123', 'Azerty123'),
             'unix_socket' => env('DB_SOCKET', ''),
```

Comment:
These changes should be made in your .env file, they are different for every developer